networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -104,8 +100,8 @@ where
 					let serialized_aptos_transaction = serde_json::to_vec(&transaction)?;
```

Comment:
We want to trace the transaction here as well, so that we know when it's come out of the mempool.